src/binary-reversal/index.js

```diff
@@ -3,6 +3,19 @@
  *
  *  * @param {string} value
  */
-function binaryReversal(value) {}
+function binaryReversal(value)
+{
+  const num = parseInt(value);
+  let binaryStr = num.toString(2);
+
+  if (binaryStr.length % 8 !== 0)
+  {
+    const fullLength = 8 - (binaryStr.length % 8) + binaryStr.length;
```

Comment:
Technically, you don't need the conditional check if you're going to use `padStart`